docs/CONFIG-VARS.md

```diff
@@ -23,7 +23,7 @@ Terraform input variables can be set in the following ways:
 
 | Name | Description | Type | Default | Notes |
 | :--- | :--- | :--- | :--- | :--- |
-| prefix | A prefix used in the name of all the GCP resources created by this script. | string | | The prefix string must start with a lowercase letter and contain only lowercase alphanumeric characters and dashes (-), but cannot end with a dash. |
+| prefix | A prefix used in the name of all the GCP resources created by this script. | string | | The prefix string must start with a lowercase letter and contain only alphanumeric characters and dashes (-), but cannot end with a dash. |
```

Comment:
Not sure if you've done a recent pull but the lowercase value is required. This was added with the prefix checks needed for kubernetes items being added that include the prefix which can only be lowercase. This needs to be added back in.

Reply:
sorry about that. Got thrown because my local was still based on pre-squash .